rtsp.go

```diff
@@ -399,7 +470,95 @@ func (rc *rtspCamera) initMJPEG(tracks media.Medias, baseURL *url.URL) error {
 
 		rc.latestFrame.Store(&img)
 	})
+	return nil
+}
+
+// SubscribeRTP registers the PacketCallback which will be called when there are new packets.
+// NOTE: Packets may be dropped before calling packetsCB if the rate new packets are received by
+// the rtppassthrough.Source is greater than the rate the subscriber consumes them.
```

Comment:
Have we been able to create this condition? Can the decoder on the webapp side recover from missing packets once passed through?